src/code/credits.asm

```diff
@@ -1641,36 +1641,54 @@ jr_017_5176:
 jr_017_51A6:
     ret                                           ; $51A6: $C9
 
-; Palette data
-Data_017_51A7::
+; Windfish palette data.
+; These are palettes for the whole fade from black to final colors,
+; for all background GBC palettes.
+; Full color is first, and darkest is last.
+WindfishPalettes::
```

Comment:
Might want to add that this palette gets loaded just before the windfish is shown. The "clouds" during the stair climb use a different palette

Reply:
Yup, I'm starting to understand how the cloud palettes get faded from white. Looks like there's a generic palettes blending function for credits.